code/modules/modular_computers/computers/machinery/modular_console.dm

```diff
@@ -59,20 +60,19 @@
 		cpu.screen_on = 1
 	update_icon()
 
-// Same as parent code, but without the smoothing handling
-/obj/machinery/modular_computer/update_icon()
-	cut_overlays()
+/obj/machinery/modular_computer/console/Destroy()
```

Comment:
Yeah... this was probably the more intelligent way of doing this rather than copy_pasting it.

Oops.